tests/conditional_compilation/tools/infer_tool.py

```diff
@@ -58,7 +60,7 @@ def cli_parser():
     :return: ir path, device and output folder path variables.
     """
     parser = argparse.ArgumentParser(description='Arguments for python API inference')
-    parser.add_argument('-m', dest='ir_path', required=True, help='Path to XML file of IR')
+    parser.add_argument('-m', dest='ir_path', required=True, help='Path to XML file of IR', nargs='+')
```

Comment:
when use `nargs='+'`, no need to parse several values via  `parse_ir_list`, parser will produce list with values.
Example:
`python3 <script>.py -m abc def` => args.ir_path will contain `['abc', `def`]`

Reply:
@vurusovs it will be list with one element which contain extra symbols like:
"['/ <path>\, \ <path\']"

Reply:
@akoryach could you provide a command line how infer_tool.py is called?